cmd/project/create_samples.go

```diff
@@ -36,13 +38,48 @@ func PromptSampleSelection(ctx context.Context, clients *shared.ClientFactory, s
 		return "", err
 	}
 
-	projectType := "deno"
-	filteredRepos := filterRepos(sampleRepos, projectType)
+	projectTypes := []string{}
+	selection, err := clients.IO.SelectPrompt(ctx, "Select a language:",
+		[]string{
+			fmt.Sprintf("Bolt for JavaScript %s", style.Secondary("Node.js")),
+			fmt.Sprintf("Bolt for Python %s", style.Secondary("Python")),
+			fmt.Sprintf("Deno Slack SDK %s", style.Secondary("Deno")),
+		},
```

Comment:
praise: Great consideration for detail by choosing to match the styling of the `create` command! 🎨